contracts/Org.sol

```diff
@@ -54,40 +58,67 @@ contract Org is Administratable {
 
   /**
    * @notice Create Organization Claim
+   * @param  claimId UUID representing this claim
    * @param  fName First name of Administrator
    * @param  lName Last name of Administrator
    * @param  eMail Email contact for Organization Administrator.
    * @param  orgAdminWalletAddress Wallet address of Organization's Administrator.
    */
   function claimRequest(
+    string memory claimId,
     string memory fName,
     string memory lName,
     string memory eMail,
     address orgAdminWalletAddress
   ) public {
+    require(
```

Comment:
Note: make sure orgAdminWalletAddress is not zero when merging in M01

Reply:
It seemed reasonable to me to ignore this check on claim submission, as long as we don't allow such a malformed claim to be approved or cashed out. It would be easy to add though, could go either way honestly.

Reply:
This is resolved in M02 I just want to ensure it stays resolved... More a reminder to me that it doesn't slip through when I merge it.

Reply:
Not sure what you mean? This is M02.

Reply:
Ahh I see you probably meant M01? All good 👍